rqt_multiplot/CMakeLists.txt

```diff
@@ -318,9 +318,6 @@ install(TARGETS rqt_multiplot
 )
 
 install(PROGRAMS scripts/rqt_multiplot
-  DESTINATION ${CATKIN_GLOBAL_BIN_DESTINATION}
```

Comment:
On 14 October 2016 at 17:37, Sam <notifications@github.com> wrote:

> In rqt_multiplot/CMakeLists.txt
> <https://github.com/ethz-asl/rqt_multiplot_plugin/pull/10#discussion_r82982418>
> :
>
> > @@ -318,9 +318,6 @@ install(TARGETS rqt_multiplot
> >  )
> >
> >  install(PROGRAMS scripts/rqt_multiplot
> > -  DESTINATION ${CATKIN_GLOBAL_BIN_DESTINATION}
>
> Why is this not required? This file and the setup.py file are similar to
> the rqt_image_view. Did this change over the time, because in
> rqt_common_plugins it's about 50:50, some packages use the same 'setup' as
> you propose and some the same as this package.
>
> @stonier <https://github.com/stonier> can you give me an explanation for
> that change? And why in rqt_common_plugins there are both ways?
>

Don't take any of what follows as gospel :)

Exactly how python scripts get installed has followed a meandering path
since catkin has been installed. Some points that have come up during the
catkin design and therafter might be of interest.


   - Global installation of python scripts is lightly discouraged because
   of the risk of name conflicts in bin
   - Yet, there are a few official ones around...mostly kept I believe for
   legacy reasons, but this makes it hard to make the above a strong convention
   - @Yujin this is one of the really few conventions we ignore
      - global scripts save so much time to find/type
      - the linux world manages to live with this without much fuss, so it
      would seem to be not critical
   - Global installation from a cmake install rule cannot get into a devel
   workspace
   - Global installation from a python setup.py can get into a devel
   workspace
   - @Yujin we typically prefer the philosophy, if it's python, do it the
   python way (i.e. setup.py) but this is not a ros convention per se
   - Some time after catkin was released there was catkin_install_python
   command added that deals with multi-python schebang issues
   - There was alot of catkin/cmake iterations on all the core ros packages
   in the first few months, new ideas patterns that emerged are not
   transmitted over all the core packages (too much time to make it so), so
   you can often see different patterns here and there

Up to you how you'd like to proceed. Being pragmatic, we'd prefer to have
it available and easily accessible in a devel workspace, but we can also
live with it only getting installed in the install space. Just let me know
and I'll touch the PR as you wish.